noisemodelling-emission/src/main/java/org/noise_planet/noisemodelling/emission/directivity/PolarGraphDirectivity.java

```diff
@@ -151,7 +218,7 @@ public String generatePolarGraph(DirectivitySphere noiseSource, double frequency
         return sb.toString();
     }
 
-    public double getDwidth() {
+    /*public double getDwidth() {
```

Comment:
If this function is never used and you want to remove it, delete the code not comment it out.
Except for Java Getter/Setter, don't remove the functions even if it is not used (will be used outside of the library)